src/runtime/Makefile

```diff
@@ -0,0 +1,21 @@
+CC            = gcc
+CFLAGS        = -std=gnu99 -s -Os -D_FILE_OFFSET_BITS=64 -ffunction-sections -fdata-sections -Wl,--gc-sections -static
+LIBS          = -lsquashfuse -lsquashfuse_ll -lzstd -lz -lpthread -ldl
```

Comment:
why is -ldl here ? dl stands for "dynamic linker", it's needed for dlopen() and friends, which are used to load shared libraries at runtime (i.e. you no longer dynamically link). musl doesn't support mixing static and dynamic linking with dlopen().

Reply:
Thanks @rofl0r. Removed it.